src/App/Webhook/PdkWebhookManager.php

```diff
@@ -123,8 +124,14 @@ protected function resolveHook(string $event): HookInterface
      */
     private function getHooks(Request $request): array
     {
-        $body = json_decode($request->getContent(), true);
+        $myParcelHeader = $request->headers->get('x-myparcel-hook');
+        $body           = json_decode($request->getContent(), true);
+        $hooks          = $body['data']['hooks'] ?? [];
+
+        return array_map(static function ($hook) use ($myParcelHeader) {
+            $hook['event'] = $myParcelHeader;
```

Comment:
zoals eerder gezegd, laat de bestaande event staan en vervang die door de header als ie niet aanwezig is 

```suggestion
            $hook['event'] = hook['event'] ?? $myParcelHeader;
```